src/dotnet/Notification.UI.Blazor/NotificationUI.cs

```diff
@@ -67,9 +69,10 @@ public async Task EnsureDeviceRegistered(CancellationToken cancellationToken)
             if (_deviceId.Value != null)
                 return;
 
-        var deviceId = await DeviceTokenRetriever.GetDeviceToken(cancellationToken);
-        if (deviceId != null)
-            _ = RegisterDevice(deviceId, cancellationToken);
+        var deviceId = await DeviceTokenRetriever.GetDeviceToken(cancellationToken).ConfigureAwait(false);
+        if (deviceId != null) {
+            await RegisterDevice(deviceId, cancellationToken).ConfigureAwait(false);
```

Comment:
Why did you decide to change and await for task completion?

Reply:
1. it's already an async method.
2. Skipping async task must be logged to avoid app misbehavior and odd bugs without any logs.
3. async is already skipped in UpdateNotificationStatus